packages/ui/src/components/filters/filters-bar/actions/sorts.tsx

```diff
@@ -95,6 +95,7 @@ const SortableItem = ({
           </DropdownMenu.Root>
         </div>
 
+        {/* TODO: Design system: Find alternate */}
         <button
```

Comment:
`Sort` and `Filter` labels are currently displayed as plain html button to match the mock. We don't have matching equivalent for this. 
<img width="1202" alt="image" src="https://github.com/user-attachments/assets/1f1a8b21-68e7-474c-9335-ba729825a48e" />

cc: @cxramina @joannalye

Reply:
should these be Dropdown components, with `Dropdown.Trigger` wrapping a ghost button?

Reply:
Ghost button is giving different look, on hover it will show the background color. As per the mock, it looks like `link` variant with primary text color.

<img width="423" alt="image" src="https://github.com/user-attachments/assets/1deaad8e-dcd1-4bc6-ba20-19e90bfa69eb" />